docs/developing_iree/profiling_vulkan_gpu.md

```diff
@@ -28,7 +28,7 @@ an Android app:
 
 ```mlir
 func @dot(%lhs: tensor<2x4xf32>, %rhs: tensor<4x2xf32>) -> tensor<2x2xf32>
-  attributes { iree.vmfb.export } {
+  attributes { iree.module.export } {
```

Comment:
Whoops, bad find-replace in https://github.com/google/iree/pull/3597 :P thanks for fixing.